### DISCUSSION TITLE
Tasmota Sonoff Basic - Toggle ON / OFF

### DISCUSSION THREAD

**Discussion by Fokker-1** [tbnobody/OpenDTU#2194] [Q&A]
Hallo,

zum Auslesen meines eHz betreibe ich einen Hichi IR-Leser mit Tasmota Sonoff Basic, das läuft prima.

Aber eine Frage.
Da gibt es einen Button "Toggle" und eine Anzeige ON bzw. OFF nach Klicken des Buttons:

![Toggle](https://github.com/user-attachments/assets/b9b12caf-ca32-4a31-893c-6b8ed3b7de6f)

Leider konnte ich bisher nirgendwo die Information finden, was über das Togglen eigentlich tatsächlich ein- und ausgeschaltet wird.
Vielleicht weiß ja hier jemand dazu eine Antwort.

Schon mal vielen Dank im Voraus.

Gruß
Werner

**Comment by Omega13x** [comment]
Hat in deinem Fall keine Funktion. Ist irgendwo in den Einstellungen vergessen worden zu deaktivieren, was aber auch egal ist. Idealerweise sieht es so aus.:
![image](https://github.com/user-attachments/assets/745fb7bf-5a9e-4a12-ae11-aea04cc01d97)

**Reply by stefan123t** [reply]
Tasmota ist eine OpenSource Software u.a. für Smart Schalter der Fa. Sonoff. Die haben idR einen GPIO den man über ein Relay o.a. zum EIN/AUSschalten einer Last (Lampe, Steckdose, etc) verwenden kann.

Mit Toggle kannst Du den virtuellen Schalter umlegen.

Hat bei der Verwendung des Tasmota OS mit einem Hichi Lesekopf keine Funktion / Bedeutung. Wahrscheinlich ist auch gar kein GPIO mehr frei / definiert auf dem ESP01.

**Comment by Juergen2453** [comment]
@Fokker-1 was hat das mit openDTU zu tun ?
Warum fragst du nicht dort nach:
https://github.com/arendst/Tasmota/discussions

**Reply by Fokker-1** [reply]
> @Fokker-1 was hat das mit openDTU zu tun ? Warum fragst du nicht dort nach: https://github.com/arendst/Tasmota/discussions

... sorry, war mir so nicht bewusst - und irren ist ja menschlich ...  :-)

Gruß
Werner

**Comment by Fokker-1** [comment]
Hi,

danke Dir, dann ist das klar, super, ich konnte auch keinerlei Reaktionen feststellen.

Kannst Du mir bitte noch die Zeile in der Konsole zeigen, über die man die ESP32 Temperatur zur Anzeige bringen kann?
Ich kenne mich da nicht so aus.

Danke schon mal und Grüße
Werner

**Reply by Omega13x** [reply]
SetOption146 1

**Reply by Fokker-1** [reply]
> SetOption146 1

Vielen Dank!

Ich habe das über die Konsole bei "Edit Script" als neue Zeile eingefügt.
Leider kommt da in der Anzeige nichts.

Aber mache Dir bitte nicht so viel Arbeit damit, ist ja nicht "lebenswichtig", wäre interessant.

Gruß
Werner

**Reply by Omega13x** [reply]
Das wird einfach in der Konsole als Befehl ausgeführt. Lässt sich mit einer 0 am Ende wieder ausschalten.
![image](https://github.com/user-attachments/assets/a4a8fe9e-4e74-456f-abcd-e752e5a82a10)
Einfach in die Zeile schreiben und mit Enter ausführen. Fertig.

**Comment by Fokker-1** [comment]
Vielen Dank.
Das habe ich so gemacht, aber es kommt keine Anzeige:

![SetOption146_1](https://github.com/user-attachments/assets/af5f50c9-63d7-4f77-9c5c-2c587abd61c1)

Kann es sein, dass bei "Edit Script" noch eine für diese Funktion erforderliche Zeile eingefügt werden muss?

Noch mal vielen Dank.

Gruß
Werner

**Reply by Omega13x** [reply]
Dann hast du keinen ESP32, sondern einen ESP8266 oder 8285. Die haben keinen Temperatursensor. Welchen Chip du hast siehst du in der Information.
![image](https://github.com/user-attachments/assets/b29670a4-6bb0-45ac-822a-3b3a5c682f2c)

**Comment by Fokker-1** [comment]
Daaaanke, das ist es, ich habe einen ESP8266EX!

Ja, wie schon geschrieben, mein Wissen um diese Dinge hat sehr viel Luft nach oben!

Dann ist jetzt alles geklärt.

Noch mal danke und Grüße
Werner

**Comment by Fokker-1** [comment]
Hallo,

zur abschließenden Info, das Problem mit Toggel ON / OFF wurde hier gelöst:

https://github.com/arendst/Tasmota/discussions/21928

Gruß
Werner
